### PR TITLE
Add shop.brendly.hr

### PR DESCRIPTION
**Pull request by dulerad94**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====
Organization Website: https://brendly.hr

My name is Dusan, I'm a CTO at Brendly.

Brendly is an e-commerce organization. We provide merchants with tools and services to sell their products online, with the ultimate goal of making commerce better for everyone.

Organization Website: https://brendly.hr

Reason for PSL Inclusion
====
In addition to allowing merchants to purchase and/or connect their domain name to their shops, we provision all new merchants with a .shop.brendly.hr subdomain. Because of this, requests between shops using.shop.brendly.hr domains are considered to be samesite, leaving them open to CSRF.

We're looking to add shop.brendly.hr to the Public Suffix List to improve cookie security for all of our merchants who do not yet make use of a custom domain.

We already added *.shop.brendly.rs 2 years ago, and now we expanding our service to Croatian market, so we need the same thing with .hr domain as we did for .rs domain.

Previous pull request with the same request: https://github.com/publicsuffix/list/pull/1309

Number of users this request is being made to serve: currently 1000, up to 50000

DNS Verification via dig
=======
dig +short TXT _psl.shop.brendly.hr
"https://github.com/publicsuffix/list/pull/1762"

Results of Syntax Checker (`make test`)
=========
Tests passed. 

-n test_NFKC:
OK
-n test_allowedchars:
OK
-n test_dots:
OK
-n test_duplicate:
OK
-n test_exception:
OK
-n test_punycode:
OK
-n test_section1:
OK
-n test_section2:
OK
-n test_section3:
OK
-n test_section4:
OK
-n test_spaces:
OK
-n test_wildcard:
OK

### CODE REVIEW

**Comment by dulerad94** [comment]
We are not sure what this check list item mean: 

"We are listing any third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see https://github.com/publicsuffix/list/issues/1245 as a well-documented example)"

As we do not seek to work around anything, should we leave this item unchecked? Thanks!

**Comment by dulerad94** [comment]
Hi,

Our domain is valid until May 2024 but has an automatic renewal every year. Our provider doesn't sell licences for more then one year. Is that OK?

**Comment by simon-friedberger** [comment]
Hi Dusan, could you please fill in the website and user numbers at the top?

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [ ] brendly.hr expires 2024-05-16 ⚠️
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [ ] Sorting
* [ ] Reasoning/Organization description

**Comment by dulerad94** [comment]
Hi Simon, 

I have updated the website URL, but what do you mean by user numbers?

**Comment by simon-friedberger** [comment]
> Number of users this request is being made to serve:

**Comment by dulerad94** [comment]
I have edited the info

**Comment by dulerad94** [comment]
I have edited the sorting

File: public_suffix_list.dat [hunk 1]
```diff
@@ -10525,6 +10525,7 @@ square7.net
 
 // Brendly : https://brendly.rs
 // Submitted by Dusan Radovanovic <dusan.radovanovic@brendly.rs>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:10527]: Please use a non-personal e-mail address.

Comment (dulerad94) [reply public_suffix_list.dat]: I have changed

**Pull request merged**